server/clip_server/model/tokenization.py

```diff
@@ -0,0 +1,82 @@
+import torch
+from typing import List, Union
+from clip_server.model.pretrained_models import _MULTILINGUALCLIP_MODELS
+
+
+class Tokenizer:
+    def __init__(self, name: str, **kwargs):
+        self._name = name
+        if name in _MULTILINGUALCLIP_MODELS:
+            import transformers
+
+            self._tokenizer = transformers.AutoTokenizer.from_pretrained(name)
+        else:
+            from clip_server.model.simple_tokenizer import SimpleTokenizer
+
+            self._tokenizer = SimpleTokenizer()
+
+    def encode(
```

Comment:
I suggest using a different name than `encode`, maybe `tokenize`, to distinguish from that in clip model

Reply:
agree

Reply:
done